suite-native/transactions/src/types.ts

```diff
@@ -0,0 +1,6 @@
+export type AddressesType = 'inputs' | 'outputs';
```

Comment:
Don't we have this type already somewhere?

Reply:
We had it defined in `transactions/selectors.ts` and I just moved it to this new file.